<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GSS Control Station - Alpha-0.1</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../gss_resources.qrc">
    <normaloff>:/files/images_icons/apps/globe.ico</normaloff>:/files/images_icons/apps/globe.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="comOpenCloseButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>350</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="HLed" name="comPortStatusLued" native="true">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>350</y>
      <width>25</width>
      <height>23</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 127);</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="receiveText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>381</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="sendText">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>60</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_settings"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuCommunication">
    <property name="title">
     <string>Communication</string>
    </property>
    <addaction name="actionCom_Settings"/>
    <addaction name="actionConnect"/>
   </widget>
   <widget class="QMenu" name="menuPerspective">
    <property name="title">
     <string>Perspective</string>
    </property>
    <addaction name="actionEngineer"/>
    <addaction name="actionOperator"/>
   </widget>
   <widget class="QMenu" name="menuMain_Widgets">
    <property name="title">
     <string>Main Widgets</string>
    </property>
    <addaction name="actionSystem_Config"/>
   </widget>
   <widget class="QMenu" name="menuTool_Widgets">
    <property name="title">
     <string>Tool Widgets</string>
    </property>
    <addaction name="actionDebug_console"/>
    <addaction name="actionParameters_Monitors"/>
    <addaction name="actionSystem_Status"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_GCS"/>
    <addaction name="actionContact"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCommunication"/>
   <addaction name="menuPerspective"/>
   <addaction name="menuMain_Widgets"/>
   <addaction name="menuTool_Widgets"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionEngineer">
   <property name="icon">
    <iconset resource="../../gss_resources.qrc">
     <normaloff>:/files/images_icons/apps/preferences_system.ico</normaloff>:/files/images_icons/apps/preferences_system.ico</iconset>
   </property>
   <property name="text">
    <string>Engineer</string>
   </property>
  </action>
  <action name="actionOperator">
   <property name="icon">
    <iconset resource="../../gss_resources.qrc">
     <normaloff>:/files/images_icons/apps/engineer_avatar.ico</normaloff>:/files/images_icons/apps/engineer_avatar.ico</iconset>
   </property>
   <property name="text">
    <string>Operator</string>
   </property>
  </action>
  <action name="actionSystem_Config">
   <property name="icon">
    <iconset resource="../../gss_resources.qrc">
     <normaloff>:/files/images_icons/apps/applications-system.png</normaloff>:/files/images_icons/apps/applications-system.png</iconset>
   </property>
   <property name="text">
    <string>System Config</string>
   </property>
  </action>
  <action name="actionCom_Settings">
   <property name="icon">
    <iconset resource="../../gss_resources.qrc">
     <normaloff>:/files/images_icons/apps/globe.ico</normaloff>:/files/images_icons/apps/globe.ico</iconset>
   </property>
   <property name="text">
    <string>Com Settings</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="../../gss_resources.qrc">
     <normaloff>:/files/images_icons/apps/disconnect.ico</normaloff>
     <normalon>:/files/images_icons/apps/connect.ico</normalon>:/files/images_icons/apps/disconnect.ico</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDebug_console">
   <property name="icon">
    <iconset resource="../../gss_resources.qrc">
     <normaloff>:/files/images_icons/apps/ultilities-terminal.png</normaloff>:/files/images_icons/apps/ultilities-terminal.png</iconset>
   </property>
   <property name="text">
    <string>Debug console</string>
   </property>
  </action>
  <action name="actionParameters_Monitors">
   <property name="icon">
    <iconset resource="../../gss_resources.qrc">
     <normaloff>:/files/images_icons/apps/utilities-system-monitor.png</normaloff>:/files/images_icons/apps/utilities-system-monitor.png</iconset>
   </property>
   <property name="text">
    <string>Parameters Monitors</string>
   </property>
  </action>
  <action name="actionSystem_Status">
   <property name="text">
    <string>System Status</string>
   </property>
  </action>
  <action name="actionAbout_GCS">
   <property name="icon">
    <iconset resource="../../gss_resources.qrc">
     <normaloff>:/files/images_icons/apps/help-browser.png</normaloff>:/files/images_icons/apps/help-browser.png</iconset>
   </property>
   <property name="text">
    <string>About GCS</string>
   </property>
  </action>
  <action name="actionContact">
   <property name="icon">
    <iconset resource="../../gss_resources.qrc">
     <normaloff>:/files/images_icons/apps/info_box_blue.ico</normaloff>:/files/images_icons/apps/info_box_blue.ico</iconset>
   </property>
   <property name="text">
    <string>Contact</string>
   </property>
  </action>
  <action name="actionLoad_settings">
   <property name="icon">
    <iconset resource="../../gss_resources.qrc">
     <normaloff>:/files/images_icons/apps/download.ico</normaloff>:/files/images_icons/apps/download.ico</iconset>
   </property>
   <property name="text">
    <string>Load settings</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../../gss_resources.qrc">
     <normaloff>:/files/images_icons/apps/process-stop.png</normaloff>:/files/images_icons/apps/process-stop.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>HLed</class>
   <extends>QWidget</extends>
   <header>hled.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../gss_resources.qrc"/>
 </resources>
 <connections/>
</ui>
